include/class.format.php

```diff
@@ -514,13 +514,21 @@ function __formatDate($timestamp, $format, $fromDb, $dayType, $timeType,
 
         if (!$timestamp)
             return '';
+	
+	// We need this codepiece to make sure we have our timestamp in 'Y-m-d H:i:s' format, as sometimes it is given to this function as unixtime
+	//TODO fix that!
+	if (is_numeric($timestamp)) {
+		$tmp = new DateTime();
+		$timestamp = $tmp->setTimestamp($timestamp)->format('Y-m-d H:i:s');
```

Comment:
It seems like this needs to be added to the next release, even if it's not perfect yet. It's a whole lot closer than the completely broken timezone assignment it was doing before. How does that happen (that this fix gets merged into the next release)?